distributed/cli/dask_worker.py

```diff
@@ -47,9 +48,21 @@
               help="Bokeh port, defaults to 8789")
 @click.option('--bokeh/--no-bokeh', 'bokeh', default=True, show_default=True,
               required=False, help="Launch Bokeh Web UI")
+@click.option('--listen-addr', type=str, default=None,
+        help="The <address>:<port> on which the worker binds to. "
+                   "The options --worker-port and --host are not compatible "
+                   "with this option. Cannot be used with --nprocs>1")
+@click.option('--contact-addr', type=str, default=None,
+        help="The <address>:<port> the worker advertises to the scheduler "
+                   "for communication with it and other workers. For eg., "
+                   "in the case of using workers behind NAT. If specified "
+                   "must also specify listen-address. "
```

Comment:
Should these helpstrings include protocol, like tcp or tls (or other)?  

Should we expand the name to `--listen-address` rather than `--listen-addr`?

I would remove the reference to the example case of a NAT in the helpstring.  This seems to be specific to a single case and not necessarily representative.